assets/assets.go

```diff
@@ -0,0 +1,31 @@
+package assets
+
+import (
+	"embed"
+	"strings"
+)
+
+//go:embed *
```

Comment:
won't this need golang 1.16?   I'm getting error while building the images. 

```assets/assets.go:4:2: cannot find package "." in:
	/go/src/github.com/openshift/local-storage-operator/vendor/embed
make: *** [Makefile:228: build-operator] Error 1```

Reply:
Yes, it requires 1.16. I check `go.mod` and it already has 1.16 listed:
https://github.com/openshift/local-storage-operator/blob/8feb36c44af009fae1fc9bf18b525bcbfc536e1d/go.mod#L3
Are there any other places that should be updated?

Reply:
I think we will have the Dockerfile

Reply:
Good catch, updated.